lib/logstash/outputs/influxdb.rb

```diff
@@ -84,15 +105,17 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   # near-real-time.
   config :idle_flush_time, :validate => :number, :default => 1
 
+
   public
   def register
-    require "ftw" # gem ftw
+    require 'manticore'
     require 'cgi'
-    @agent = FTW::Agent.new
+    
+    @client = Manticore::Client.new
     @queue = []
 
-    @query_params = "u=#{@user}&p=#{@password.value}&time_precision=#{@time_precision}"
-    @base_url = "http://#{@host}:#{@port}/db/#{@db}/series"
+    @query_params = "db=#{@db}&rp=#{@retention_policy}&precision=#{@time_precision}&u=#{@user}&p=#{@password.value}"
```

Comment:
both user and password are made optional in this PR, so `@user` and `@password.value` can be nil here. Is that ok?